Sources/Controllers/MessagesViewController.swift

```diff
@@ -59,6 +61,18 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
         return false
     }
 
+    /// A CGFloat value that adds to (or, if negative, subtracts from) the automatically
+    /// computed value of `messagesCollectionView.contentInset.bottom`. Meant to be used
+    /// as a measure of last resort when the built-in algorithm does not produce the right
+    /// value for your app. Please let us know when you end up having to use this property.
+    open var extraBottomInset: CGFloat = 0 {
+        didSet {
+            let delta = extraBottomInset - oldValue
+            guard abs(delta) > coordinatePrecision else { return }
+            messageCollectionViewBottomInset += delta
```

Comment:
What's the reasoning behind not just applying the `newValue` to the bottom inset directly? I'm concerned this is unexpected behavior

Reply:
@SD10 Because the newValue here is the _additional_ bottom inset, not the entire bottom inset. :-)